Makefile

```diff
@@ -1,18 +1,20 @@
 # Set DESTDIR if it isn't given
 DESTDIR?=/
 
-.PHONY : all clean clean-build clean-pyc clean-test install test upload
+.PHONY : all clean clean-build clean-pyc clean-test install test upload develop
 
 all :
 	python ./setup.py build
 
 install : all
 	python ./setup.py install --root=${DESTDIR}
 
+develop :
```

Comment:
Is this target needed ?

python setup.py develop will already install dependency for *running* paho-mqtt (currently nothing).
It only add dependency needed to run test, which is handled by python setup.py test.